src/components/facet-line/FacetLineSelected.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import {
+  FilterItemTerm,
+  TermOnClickHandler
+} from "../../apps/search-result/types";
+import ButtonTag from "../Buttons/ButtonTag";
+
+type FacetLineSelectedProps = {
+  filters: { [key: string]: { [key: string]: FilterItemTerm } };
+  filterHandler: TermOnClickHandler;
+};
+
+const FacetLineSelected: React.FunctionComponent<FacetLineSelectedProps> = ({
+  filters,
+  filterHandler
+}) => {
+  return (
+    <ul className="facet-line-selected">
+      {Object.entries(filters).map(([facet, value]) => {
+        return (
+          <>
+            {Object.entries(value).map(([label, term]) => {
+              return (
+                <li className="facet-line-selected-terms">
```

Comment:
It is outside the scope of this PR but I would have expected the terms to be BEM elements within the selected block e.g. `facet-line-selected__term`.